Epona/include/Epona/Debug.hpp

```diff
@@ -1,5 +1,5 @@
 /*
-* Copyright (C) 2018 by Godlike
+* Copyright (C) 2019 by Godlike
```

Comment:
why this isolated change? please revert if nothing in file is changed - such changes introduce misleading `git blame` and `git log` behavior